File: Python/1. Introduction/exercise6.py
# Write a function "https://www.hackerrank.com/challenges/write-a-function/problem"


def is_leap(year):
    leap = False

    # Write your logic here
    if (year % 4 == 0) and ((year % 100 != 0) or (year % 400 == 0)):
        if (year % 4 == 0):
            leap = True
    return leap

File: Java/1. Introduction/exercise6.java
// Java Loops II "https://www.hackerrank.com/challenges/java-loops/problem"

import java.util.Scanner;


class exercise6{

    public static void main(String []arg){
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();

        for (int i=0; i<t; i++) {
            int a = in.nextInt();
            int b = in.nextInt();
            int n = in.nextInt();
            int sum = 0;

            for (int j=0; j<n; j++) {
                sum += (Math.pow(2, j) * b);
                System.out.print((sum+a) + " ");
            }
        System.out.println();
        }
        in.close();
    }
}
